pages/moderation.vue

```diff
@@ -74,6 +74,8 @@ const router = useRouter()
 const doesTheUserHaveAccess: Ref<boolean> = ref(true)
 
 const checkIfUserIsLoggedIn = async () => {
+    // init the auth to see if we are in dev/testing mode or prod
```

Comment:
```suggestion
    // Inits the auth to see if we are in dev/testing mode or prod.
```

This is a small nit. We should determine and stick to our conventions for leaving comments in code. Like if there should always be a space after the `//` and if it should start with an uppercase and end with a period. You can see the inconsistencies in the two comments below this one. Each one is different.